apps/ledger-live-mobile/src/screens/Settings/Debug/OpenDebugFeatureFlags.js

```diff
@@ -0,0 +1,16 @@
+// @flow
```

Comment:
Shouldn't it be in typescript ?

Reply:
🤷🏼 it's a mess on LLM and I just copied the other button